.codespellrc

```diff
@@ -0,0 +1,4 @@
+[codespell]
+skip = .git,*.pdf,*.svg
+# BU - part of JP3t17pCcGlemwknint6hfoeCVQrEMVwxRLRjXpq+BU=
```

Comment:
@yarikoptic as someone who is not too familiar with codespell what does this represent?

Reply:
This string is in a `go.sum` file. Does it not make more sense to add `*.sum` to the `skip` list?

Reply:
indeed, done!